Set/Player/Player.swift

```diff
@@ -0,0 +1,7 @@
+import Foundation
+
+struct Player{
+    var name: String
+    var score: Int
+    var isPlaying = false
```

Comment:
Honestly, I don't really like that `Player` model knows something about his turn in the game. I would add `id: UUID` to the model and `GameModel` than keep tracking of currently active player by `Player.id`.

Reply:
fixed